ci_framework/roles/hive/tasks/oc_delete.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# Removes the resource from OpenShift Cluster
+-
+  name: "Deleting {{ resource }}"
```

Comment:
Can we merge these two lines so that it is
```
- name: "Deleting {{ resource_type }}/{{ resource }}" 
```

Reply:
ack, changing to ```name: "Deleting {{ resource }}/{{ resource_name }}"```